Provide instructions on Agent minor version upgrades (#13224)
DataDog/documentation: content/en/agent/versions/upgrade_between_agent_minor_versions.md

---
title: Upgrade Between Datadog Agent Minor Versions
kind: documentation
aliases:
  - /agent/faq/upgrade-between-agent-minor-versions
  - /agent/guide/upgrade-between-agent-minor-versions
---

> ## Upgrade Between Minor Versions of Agent 6 and 7

Reviewer: ```suggestion
## Upgrade between minor versions of Agent 6 and 7
```

{{< tabs >}}
{{% tab "Linux" %}}

> The recommended way to upgrade between minor versions of Agent 6 and 7 is using the `install_script.sh` script. Following commands work on all supported Linux distributions.

Reviewer: ```suggestion
The recommended way to upgrade between minor versions of Agent 6 and 7 is to use the `install_script.sh` script. The following commands work on all supported Linux distributions.
```

Upgrading to a given Agent 6 minor version:

: `DD_AGENT_MAJOR_VERSION=6 DD_AGENT_MINOR_VERSION=<target_minor> bash -c "$(curl -L https://s3.amazonaws.com/dd-agent/scripts/install_script.sh)"`

Upgrading to the latest Agent 6 minor version:

: `DD_AGENT_MAJOR_VERSION=6 bash -c "$(curl -L https://s3.amazonaws.com/dd-agent/scripts/install_script.sh)"`

Upgrading to a given Agent 7 minor version:

: `DD_AGENT_MAJOR_VERSION=7 DD_AGENT_MINOR_VERSION=<target_minor> bash -c "$(curl -L https://s3.amazonaws.com/dd-agent/scripts/install_script.sh)"`

Upgrading to the latest Agent 7 minor version:

> : `DD_AGENT_MAJOR_VERSION=7 bash -c "$(curl -L https://s3.amazonaws.com/dd-agent/scripts/install_script.sh)"`

Author: I would like to keep this explicit, in case we decide to change the default at some point.

{{% /tab %}}
{{% tab "Windows" %}}

> Download and install the desired installation package.

Reviewer: ```suggestion
Download and install the specific version's installation package.
```

> URL for a given minor version of Agent 6:

Reviewer: ```suggestion
URL to download a specific Agent 6 minor version:
```

: `https://ddagent-windows-stable.s3.amazonaws.com/ddagent-cli-6.<minor_version>.<bugfix_version>.msi`

> URL for a given minor version of Agent 7:

Reviewer: ```suggestion
URL to download a specific Agent 7 minor version
```

: `https://ddagent-windows-stable.s3.amazonaws.com/ddagent-cli-7.<minor_version>.<bugfix_version>.msi`

{{% /tab %}}
{{% tab "MacOS" %}}

> **Note**: There is currently no way to upgrade to a specific minor version.

Reviewer: ```suggestion
**Note**: It is not possible to upgrade to a specific minor version.
```

> Upgrading to the latest Agent 6 minor version:

Reviewer: ```suggestion
Command to upgrade to the latest Agent 6 minor version:
```

: `DD_AGENT_MAJOR_VERSION=6 bash -c "$(curl -L https://dd-agent.s3.amazonaws.com/scripts/install_mac_os.sh)"`

> Upgrading to the latest Agent 7 minor version:

Reviewer: ```suggestion
Command to upgrade to the latest Agent 7 minor version:
```
